projects/opendr_ws_2/src/data_generation/data_generation/synthetic_facial_generation.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env python3.6
+# Copyright 2020-2022 OpenDR European Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import cv2
+import torch
+import rclpy
+from ros2_bridge.bridge import ROS2Bridge
+import numpy as np
+from sensor_msgs.msg import Image as ROS_Image
+from SyntheticDataGeneration import MultiviewDataGeneration
```

Comment:
This import does not seem to work, the algorithm is placed in `projects` instead of (probably) in `src`. A regular opendr import in a ROS node should be something like `from opendr.perception.pose_estimation import LightweightOpenPoseLearner`, so `from opendr.data_generation.synthetic_facial_generation import SyntheticFacialGenerationLearner` or something similar.

This should need to be resolved outside this PR. Right now i am not able to run the node locally even with other proposed changes, due to this import as well as the import in line 26 below.